app/_src/networking/transparent-proxy/kubernetes.md

```diff
@@ -0,0 +1,265 @@
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+### Control Plane Runtime Configuration
+<!-- vale Google.Headings = YES -->
+
+The control plane runtime configuration is a section of {{ Kuma }}'s configuration that contains important settings for how the control plane operates, including options for the transparent proxy and other key components.
+
+Some transparent proxy settings can be adjusted here, and **for certain settings, this is the only place they can be modified**.
+
+Currently, it’s best to use the control plane runtime configuration as the main place to store transparent proxy settings that apply to all [workloads](https://kubernetes.io/docs/concepts/workloads/). In the future, once the [Configuration in ConfigMap](#configuration-in-configmap-experimental) feature is fully supported, we’ll recommend using ConfigMaps for these settings. However, a few restricted settings that are rarely customized will still need to be in the control plane’s runtime configuration.
+
+{% tip %}
+For more details, see the [Control Plane Configuration Reference]({{ docs }}/reference/kuma-cp/) under `runtime.kubernetes`. You can also find transparent proxy-specific settings in the [Control Plane Runtime Configuration section]({{ docs }}/reference/transparent-proxy-configuration/#control-plane-runtime-configuration) of the Transparent Proxy Configuration reference.
+{% endtip %}
+
+#### Settings restricted to control plane runtime configuration
+
+Some transparent proxy settings **can only be changed through the control plane’s runtime configuration** because they are shared with other {{ Kuma }} components. These settings handle essential tasks like creating [`Dataplane` resources]({{ docs }}/production/dp-config/dpp/) for workloads and setting up the [`kuma-dp` sidecar container]({{ docs }}/production/dp-config/dpp-on-kubernetes/) alongside them. For example, the DNS port used to redirect traffic is shared between the transparent proxy and the `kuma-dp run` command in the sidecar container. Keeping these settings consistent across workloads prevents misconfigurations. These settings are:
+
+- `runtime.kubernetes.injector.sidecarContainer.redirectPortInbound`
+- `runtime.kubernetes.injector.sidecarContainer.redirectPortOutbound`
+- `runtime.kubernetes.injector.sidecarContainer.uid`
+- `runtime.kubernetes.injector.builtinDNS.enabled`
+- `runtime.kubernetes.injector.builtinDNS.port`
+
+{% danger %}
+⚠️ IMPORTANT
+{:.custom-block-title}
+
+If you change these settings, but with values different from the runtime configuration, they will be ignored, and a warning will appear in the control plane logs (except for DNS-related settings).
```

Comment:
If you change these settings by Annotation or ConfigMap the change will be ignored, and a warning will appear in the control plane logs (except for DNS-related settings).